get_instagram.py

```diff
@@ -0,0 +1,29 @@
+from InstagramAPI import InstagramAPI
+from key import user_name, password
+
+
+class GetInstagramInfo:
+    def __init__(self, username, password):
+	self.username = username
```

Comment:
I don't think we really need these fields.

Reply:
it is true